src/background.ts

```diff
@@ -48,6 +52,22 @@ function setIconAndPopup(active: string, tabId: number) {
   chrome.pageAction.show(tabId);
 }
 
+// Detect what storefront backend is used. Logic taken from Ubercorn.
+if (typeof chrome.extension !== 'undefined') {
```

Comment:
I don't think this check is needed since this extension is only targetting Chrome and not other browsers? If it is needed, we need to do it everywhere else

Reply:
It was to prevent an error when loading the extension in the Extensions page for Chrome. `chrome.webRequest` is not available on that page. Maybe I should check for `typeof chrome.webRequest !== 'undefined'` instead? Would make this clearer.